app/pages/solutions-list/context.js

```diff
@@ -1,17 +1,25 @@
-const createSolutionsContext = (filterType, solutions) => solutions.map(solution => ({
-  id: solution.id,
-  name: solution.name,
-  summary: solution.summary,
-  supplierName: solution.supplier && solution.supplier.name,
-  capabilities: solution.capabilities && solution.capabilities.map(capability => capability.name),
-  isFoundation: solution.isFoundation,
-  viewSolutionUrl: `/solutions/${filterType}/${solution.id}`,
-}));
+const createSolutionsContext = (filterType, solutions, query) => solutions.map((solution) => {
+  const viewSolutionUrl = filterType === 'capabilities-selector'
+    ? `/solutions/capabilities-selector/selected/${solution.id}?${query}`
+    : `/solutions/${filterType}/selected/${solution.id}`;
+  return ({
+    id: solution.id,
+    name: solution.name,
+    summary: solution.summary,
+    supplierName: solution.supplier && solution.supplier.name,
+    capabilities: solution.capabilities && solution.capabilities.map(capability => capability.name),
+    isFoundation: solution.isFoundation,
+    viewSolutionUrl,
+  });
+});
 
 export const createSolutionListPageContext = ({
-  filterType, solutionListManifest, solutionsData,
+  filterType, solutionListManifest, solutionsData, query,

```

Comment:
can we name query to be something more specific like `selectedCapabilitiesQuery`